src/components/Farm/FarmPoolTable.tsx

```diff
@@ -76,15 +76,15 @@ const FarmPoolTable = ({ poolsInfo, v0PoolsInfo, v1PoolsInfo, tokenPrice, select
                     <TYPE.thHeader style={{ justifySelf: 'flex-start' }}>{t('apr')}</TYPE.thHeader>
                   </RowFixed>
                   <RowFixed width="20%">
-                    <TYPE.thHeader style={{ justifySelf: 'flex-start' }}>{t('totalValueLocked')}</TYPE.thHeader>
+                    <TYPE.thHeader style={{ justifySelf: 'flex-start' }}>{t('totalValueStaked')}</TYPE.thHeader>
                   </RowFixed>
                   <RowFixed width="13%">
                     <TYPE.thHeader style={{ justifySelf: 'flex-start' }}>{t('stakeable')}</TYPE.thHeader>
                   </RowFixed>
                   <RowFixed width="16%">
                     <TYPE.thHeader style={{ justifySelf: 'flex-start' }}>{t('valueStaked')}</TYPE.thHeader>
                   </RowFixed>
-                  <RowFixed width="16%">
+                  <RowFixed width="20%">
```

Comment:
@geonellov width has been changed here but no further adjustment on other `RowFixed` siblings, the sum of the siblings should be 100%

Reply:
@schystz fixed.